### PR TITLE
VMO-7146 Remove default exit label overrides

### PR DESCRIPTION
**Pull request by object-required**
See screenshots in the [Voto5 PR #2793](https://bitbucket.org/voto/voto5/pull-requests/2793/vmo-7146-default-exit-names).

Screen Shot 2022-10-18 at 1 31 49 PM

### CODE REVIEW

File: builder.config.json [hunk 1]
```diff
@@ -200,9 +194,6 @@
           "primary": {
             "name": "Valid",
             "test": "@ISNUMBER(block.value)"
-          },
-          "default": {
-            "name": "Invalid"
```
Comment (safydy-r) [inline builder.config.json:205]: @object-required , As discussed, I'll put my review on hold until we have a clarification from Nancy what she mean about 
-`The final exit for every block type (in every mode) should be named “Default”`
- `This is true for most block right now, but should be updated for`
 (especially for Numeric block)

**Review by bulatgab — APPROVED** [review]
LGTM 👍

File: builder.config.json (hunk 1 above)
Comment (safydy-r) [reply builder.config.json:205]: This is resolved after our standup meeting

**Review by safydy-r — APPROVED** [review]
I think this is good now, thanks

**Pull request merged**